### PR TITLE
fix: fix hero animation not being paused when offscreen

### PR DESCRIPTION
**Pull request by satazor**
Refs #69.

### CODE REVIEW

File: src/shared/components/hero-section/index.js [hunk 1]
```diff
@@ -58,12 +58,12 @@ class Hero extends Component {
...
       [styles.error]: !isDataLoaded && existsError
     })
-    const observerClass = classNames({ [styles.animationOff]: !inView })
+    const wrapperContainerClasses = classNames(styles.wrapperContainer, { [styles.animationOff]: !inView })
     const messages = this.messages
 
     return (
-      <Observer onChange={ this.handleObserverView } className={ observerClass }>
-        <div className={ styles.wrapperContainer }>
+      <Observer onChange={ this.handleObserverChange }>
```
Comment (vasco-santos) [inline src/shared/components/hero-section/index.js]: Something is strange here. You forgot to change the function name below

```
Warning: Failed prop type: The prop `onChange` is marked as required in `IntersectionObserver`, but its value is `undefined`
```

Comment (vasco-santos) [reply src/shared/components/hero-section/index.js]: We can wait for [js.ipfs.io#77](https://github.com/ipfs/js.ipfs.io/pull/77/) being merged.

This fix happens here:
https://github.com/ipfs/js.ipfs.io/pull/77/files#diff-1370ae5a38b12fa05367f677a13fec38R111

Comment (satazor) [reply src/shared/components/hero-section/index.js]: Should be fixed

Comment (satazor) [reply src/shared/components/hero-section/index.js]: @vasco-santos you can merge #77, I will rebase afterwards

Comment (vasco-santos) [reply src/shared/components/hero-section/index.js]: done!

**Pull request merged**